src/Faker/Provider/ar_SA/Internet.php

```diff
@@ -11,18 +11,22 @@ class Internet extends \Faker\Provider\Internet
         '?{{lastNameAscii}}',
     );
     protected static $safeEmailTld = array(
-        'com', 'jo', 'me', 'net', 'org',
+        'com', 'jo', 'me', 'net', 'org', 'com.eg',
     );
 
     protected static $tld = array(
-        'biz', 'com', 'info', 'sa', 'net', 'org',
+        'biz', 'com', 'info', 'sa', 'net', 'org', 'com.eg',
     );
 
     protected static $lastNameAscii = array(
-        'abbad', 'abbadi', 'abbas', 'abulebbeh', 'flefel', 'hadi', 'hamad', 'hasan', 'jabri', 'kanaan', 'karam', 'maanee', 'melhem', 'nimry', 'obaisi', 'qasem', 'qawasmee', 'rabee', 'rashwani', 'shami', 'zaloum',
+        'abbad', 'abbadi', 'abbas', 'abulebbeh', 'flefel', 'hadi', 'hamad', 'hasan', 'jabri', 'kanaan', 'karam', 'maanee', 'melhem',
```

Comment:
Where does this list come from? Please quote the source in the comments. Same for the first names.